edb/schema/schema.py

```diff
@@ -1028,3 +1033,43 @@ def _get_operators(
         Tuple[s_oper.Operator, ...],
         tuple(schema._id_to_type[oid] for oid in objids),
     )
+
+
+@functools.lru_cache()
```

Comment:
I'd limit the size of this cache. Would 10 work?

Reply:
Why so small?  Schema objects are lightweight (`PyObject header` + `uuid` basically), so the default limit shouldn't eat away at the memory too much.

Reply:
Schema objects are small but they hold references to potentially large trees of other objects. So my question still stands: would a small number like 10 work? How often is this function called?